components/Home/Bonding/TokenManagement/WsolDeposit.jsx

```diff
@@ -0,0 +1,177 @@
+import { useEffect, useState } from 'react';
+import {
+  Button, Form, InputNumber, Flex, Typography, Spin, Alert,
+} from 'antd';
+import pDebounce from 'p-debounce';
+import { isNumber } from 'lodash';
+import { LAMPORTS_PER_SOL } from '@solana/web3.js';
+import {
+  getCommaSeparatedNumber,
+  notifyError,
+} from '@autonolas/frontend-library';
+
+import { useSvmConnectivity } from 'common-util/hooks/useSvmConnectivity';
+import { useWsolDeposit } from './hooks/useWsolDeposit';
+import { DEFAULT_SLIPPAGE, slippageValidator } from './utils';
+
+const { Text } = Typography;
+
+export const WsolDeposit = () => {
+  const [form] = Form.useForm();
+  const { isSvmWalletConnected } = useSvmConnectivity();
+  const [estimatedQuote, setEstimatedQuote] = useState(null);
+  const [isEstimating, setIsEstimating] = useState(false);
+  const [isDepositing, setIsDepositing] = useState(false);
+  const [bridgedTokenAmount, setBridgedTokenAmount] = useState(null);
+
+  const {
+    getDepositIncreaseLiquidityQuote: fn,
+    getDepositTransformedQuote,
+    deposit,
+  } = useWsolDeposit();
+  const getDepositQuote = pDebounce(fn, 500);
+
+  // initially, set default slippage value
+  useEffect(() => {
+    form.setFieldsValue({ slippage: DEFAULT_SLIPPAGE });
```

Comment:
do you think we can use `initialValues` on Form or `initialValue` on Form.Item instead? https://4x.ant.design/components/form/#Difference-between-Form-initialValues-and-Item-initialValue